server/src/Config.ts

```diff
@@ -57,6 +58,7 @@ export const getConfig = (): Config => ({
 	WEB_HOOK_TOKEN: env.get('WEB_HOOK_TOKEN').default(defaultConfig.WEB_HOOK_TOKEN).asString(),
 	DRY_RUN: env.get('DRY_RUN').default(`${defaultConfig.DRY_RUN}`).asBoolStrict(),
 	HTTP_PROXY: env.get('HTTP_PROXY').default('').asString(),
+	PROJECT_ID: env.get('PROJECT_ID').default('').asString(),
```

Comment:
We have multiple repositories in our gitlab even now and the number will grown even though our main repo is a monorepo. I'd say that support for multiple project ids is a must.

Reply:
updated.